### PR TITLE
[Community Tutorials] Public Health Tutorials

### PR DESCRIPTION
**Pull request by hiyer09**
These are two tutorials I prepared for the Geo For Good 2019 Summit.

### CODE REVIEW

**Comment by hiyer09** [comment]
> googlebot

@googlebot I signed it!

**Comment by gino-m** [comment]
Thanks for the submissions! A few small requests:

- [ ] Could you kindly submit one PR for each tutorial? This way we can assign each to the appropriate technical reviewer
- [ ] If you haven't already, be sure to read through the instructions [here](https://developers.google.com/earth-engine/tutorials/community/write) and the style guide [here](https://developers.google.com/earth-engine/tutorials/community/styleguide)
- [ ] Please be sure to accept the CLA

Once these are done we'll find a technical reviewer for your content.

File: tutorials/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+author: hiyer20
+tags: comma-separated, lowercase, list, of, related, keywords
+date_published: 2019-09-17
+---
+
+
+---
+title: Earth Engine for Public Health Land Surface Temperature in Uganda Example
+
+---
```
Comment (gino-m) [inline tutorials/index.md]: Each tutorial should only have one front matter block; you can remove this duplicate one.

File: tutorials/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+[Open In Code Editor](https://code.earthengine.google.com/1a68fa019a5175a0250ecba425ae5e0d)
+
+Learning objectives:
+•	Introduce remote sensing data.
```
Comment (gino-m) [inline tutorials/index.md]: Please use Markdown `*` or `-` without additional spaces for bullet points, e.g.:
```
* Introduce..
* Learn..
```

File: tutorials/g4g-export-mult-raster/index.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+---
```
Comment (gino-m) [inline tutorials/g4g-export-mult-raster/index.md]: Please remove g4g- prefix from files.

File: tutorials/g4g-public-health-ug/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
+title: Earth Engine for Public Health Land Surface Temperature Example Uganda
```
Comment (gino-m) [inline tutorials/g4g-public-health-ug/index.md]: No need to mention Earth Engine in the title, since it will already be under EE docs.

**Comment by gino-m** [comment]
Hi @hiyer09, there are a few pending suggestions. Would you be able to take a look and/or let me know if you're still working on this? Thanks!

File: tutorials/g4g-public-health-ug/index.md (hunk 4 above)
Comment (hiyer09) [reply tutorials/g4g-public-health-ug/index.md]: Thanks for your comment - I have completed

**Comment by gino-m** [comment]
> Hi @hiyer09, there are a few pending suggestions. Would you be able to take a look and/or let me know if you're still working on this? Thanks!

@hiyer09 Gentle ping. Let me know if you have any questions!

File: tutorials/export-mult-raster/index.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+1. We first have to tell Google Earth Engine that the dates we have defined above are in fact date variables.
+2. Next, we filter on the start and end dates specified earlier, and the region object we defined by drawing a polygon around Botswana
+3. It's good practice to check how many images we get - we want to just keep one
+4. We take the earlier image by sorting on this element in the data
+5. Our function spits out a singe image in Botswana for the season we specify
+
+```
+
+// Write a function to grab image with data you need
+var GetImage = function(bdt, edt) {
```
Comment (gino-m) [inline tutorials/export-mult-raster/index.md]: Functions usually start with lowercase (`getImage()`) in JavaScript.

File: tutorials/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
+---
```
Comment (gino-m) [inline tutorials/index.md]: Did you meant to replace this file with `public-health-ug/index.md`?

**Comment by gino-m** [comment]
> A few minor editorial fixes to do while splitting PRs (i.e. create two branches in your cloned repo and commit and push). If you'd like help doing this let me know!

Could you kindly also split this into two separate PRs? That will help us split the review load between reviewers, and lead to quicker turnaround. Thanks!

File: tutorials/index.md (hunk 6 above)
Comment (hiyer09) [reply tutorials/index.md]: Yes - I thought that addressed the comment about splitting PRs/creating separate tutorials? As you can see I am new to this and am finding the process rather confusing.. sorry

**Comment by gino-m** [comment]
> I don't know how to do this, sorry, could you please direct me to a tutorial?

No worries, happy to help. Are you using the GitHub website entirely, or are you working from a commnand-line as well? Using GitHub only, you'd follow the same instructions you used to create this tutorial at https://developers.google.com/earth-engine/tutorials/community/write, pasting in the second tutorial and deleting it from this Pull Request.

Let me know how that goes and if you hit any bumps - we want to make it as easy as possible for other potential contributors as well!

File: tutorials/export-mult-raster/index.md (hunk 5 above)
Comment (hiyer09) [reply tutorials/export-mult-raster/index.md]: Thanks - is it essential to change (noting that many community users may not be familiar with Javascript function naming conventions?)

Comment (gino-m) [reply tutorials/export-mult-raster/index.md]: Yes please. Even if a user only interacts with JavaScript via Earth Engine, Community Tutorials are published in our official documentation alongside other examples, API docs, where should strive for consistency. Perhaps more importantly, capital `GetImage()` has special meaning in JavaScript (an object constructor), which if used improperly, can confuse learners of the language. 

Thanks for your patience!

**Comment by hiyer09** [comment]
I have stopped working on these, tried to copy/paste them into new tutorials using the "writing your own community tutorials." There are now separate pull requests in patch-2 and patch-3.

File: tutorials/export-mult-raster/index.md (hunk 5 above)
Comment (hiyer09) [reply tutorials/export-mult-raster/index.md]: Understood, I will change in the new version. Thanks!

**Pull request closed without merging**